app/validators/sections/section_validator.py

```diff
@@ -40,6 +41,24 @@ def validate_summary(self):
             for item in section_summary.get("items", []):
                 self.validate_list_exists(item.get("for_list"))
 
+    def validate_section_enabled(self):
+        section_enabled = self.section.get("enabled", None)
+
+        if section_enabled and isinstance(section_enabled, list):
+            for enabled in self.section.get("enabled"):
+                when = enabled["when"]
+                when_validator = WhenRuleValidator(
+                    when, self.section["id"], self.questionnaire_schema
+                )
+                self.errors += when_validator.validate()
+
+        if section_enabled and isinstance(section_enabled, dict):
+            when = self.section.get("enabled")["when"]
```

Comment:
Worth reusing the already existing var `section_enabled` instead of access again.

Reply:
Same, using the var, thanks!